test/parallel/test-tls-getprotocol.js

```diff
@@ -14,8 +16,8 @@ const clientConfigs = [
 ];
 
 const serverConfig = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent2-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent2-cert.pem`)
+  key: fs.readFileSync(`${fixturesDir}/keys/agent2-key.pem`),
```

Comment:
It turns out that reading keys from the fixtures folder is so common in the tests that there's an even more concise wrapper for it:  [`fixtures.readKey()`](https://github.com/nodejs/node/tree/master/test/common#fixturesreadkeyarg-enc).

Usage will be something like `key: fixtures.readKey('agent2-key.pem'),`

Reply:
Thank you sir... will make the updates :)!!!